### PR TITLE
feat(performance): Cell action to set Transaction thresholds

### PR DESCRIPTION
**Pull request by shruthilayaj**
Add a cell action on user misery to set transaction-level
thresholds.

Screenshots:
Screen Shot 2021-07-05 at 4 29 08 PM

Screen Shot 2021-07-05 at 4 29 15 PM

### CODE REVIEW

File: static/app/views/performance/table.tsx [hunk 1]
```diff
@@ -89,6 +100,40 @@ class Table extends React.Component<Props, State> {
...
+          modalProps => (
+            <TransactionThresholdModal
+              {...modalProps}
+              organization={organization}
+              transactionName={transactionName}
+              eventView={eventView}
+              project={projectID}
+              transactionThreshold={project_threshold[1]}
+              transactionThresholdMetric={project_threshold[0]}
+              onApply={(threshold, metric) => {
```
Comment (shruthilayaj) [inline static/app/views/performance/table.tsx:118]: I'm reloading the data on the table when the threshold is updated

File: static/app/views/performance/table.tsx [hunk 2]
```diff
@@ -89,6 +100,40 @@ class Table extends React.Component<Props, State> {
...
+                if (
+                  threshold !== project_threshold[1] ||
+                  metric !== project_threshold[0]
+                ) {
+                  this.setState({
```
Comment (k-fish) [inline static/app/views/performance/table.tsx:123]: This sets it locally to the table, how does this persist though?

Comment (k-fish) [reply static/app/views/performance/table.tsx:123]: Nvm, ignore me lol.

**Review by k-fish — APPROVED** [review]

File: tests/js/spec/views/performance/table.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+  wrapper.update();
+
+  // Open the menu
+  wrapper.find('MenuButton').simulate('click');
+
+  // Return the menu wrapper so we can interact with it.
+  return wrapper.find('CellAction').at(cellIndex).find('Menu');
+}
+
+describe('Performance > Content', function () {
```
Comment (k-fish) [inline tests/js/spec/views/performance/table.spec.jsx:42]: `Performance > Table`

**Pull request merged**